src/v2/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAdditionalInfo.tsx

```diff
@@ -8,101 +9,123 @@ import {
   Text,
 } from "@artsy/palette"
 import { ArtworkDetailsAdditionalInfo_artwork } from "v2/__generated__/ArtworkDetailsAdditionalInfo_artwork.graphql"
-import React from "react"
+import React, { useState } from "react"
 import { createFragmentContainer, graphql } from "react-relay"
 
 import { RequestConditionReportQueryRenderer } from "./RequestConditionReport"
+import { ArtworkDetailsMediumModalFragmentContainer } from "../ArtworkDetailsMediumModal"
 
 export interface ArtworkDetailsAdditionalInfoProps {
   artwork: ArtworkDetailsAdditionalInfo_artwork
 }
 
-export class ArtworkDetailsAdditionalInfo extends React.Component<
-  ArtworkDetailsAdditionalInfoProps
-> {
-  render() {
-    const {
-      category,
-      series,
-      publisher,
-      manufacturer,
-      image_rights,
-      internalID,
-      canRequestLotConditionsReport,
-      framed,
-      signatureInfo,
-      conditionDescription,
-      certificateOfAuthenticity,
-    } = this.props.artwork
+export const ArtworkDetailsAdditionalInfo: React.FC<ArtworkDetailsAdditionalInfoProps> = ({
+  artwork,
+}) => {
+  const {
+    category,
+    series,
+    publisher,
+    manufacturer,
+    image_rights,
+    internalID,
+    canRequestLotConditionsReport,
+    framed,
+    signatureInfo,
+    conditionDescription,
+    certificateOfAuthenticity,
+  } = artwork
 
-    const listItems = [
-      {
-        title: "Medium",
-        value: category,
-      },
-      {
-        title: "Condition",
-        value: canRequestLotConditionsReport ? (
-          <RequestConditionReportQueryRenderer artworkID={internalID} />
-        ) : (
-          conditionDescription && conditionDescription.details
-        ),
-      },
+  const [open, setOpen] = useState(false)
 
-      {
-        title: "Signature",
-        value: signatureInfo && signatureInfo.details,
-      },
-      {
-        title: "Certificate of authenticity",
-        value: certificateOfAuthenticity && certificateOfAuthenticity.details,
-      },
-      {
-        title: "Frame",
-        value: framed && framed.details,
-      },
-      { title: "Series", value: series },
-      { title: "Publisher", value: publisher },
-      { title: "Manufacturer", value: manufacturer },
-      { title: "Image rights", value: image_rights },
-    ]
+  const listItems = [
+    {
+      title: "Medium",
+      value: (
+        <>
+          {artwork.mediumType ? (
```

Comment:
It looks like there might be a bug in the resolver for `mediumType` as sometimes we have a `category` but no `mediumType` to correspond.

Reply:
Let me know if you come across any specific examples. Might be due to a mismatch between the `category` value and the keys in https://github.com/artsy/metaphysics/blob/master/src/lib/artworkMediums.ts

Reply:
The issue seems to be fixed now. May have just been dependent on PR from earlier hitting staging. 🤷  I was previously seeing `Drawing, Collage or other Work on Paper` with this, but looks good now.

Reply:
Yep, that case was fixed with https://github.com/artsy/metaphysics/commit/9a0c5b71ae074a4f2add034c80ce56abc2dc5f5f

I only saw that one mismatch between https://github.com/artsy/kinetic/blob/3d997e714f55cce23a6432bffe01d04d2327fbd1/app/models/kinetic/artwork.rb#L231-L251 and https://github.com/artsy/metaphysics/blob/master/src/lib/artworkMediums.ts so hopefully we're covered now!